# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../btrfs-progs/compile.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Depending on architecture and GCC version yields some warnings here and there
(like uninitialized use or long prints on 64bit CPUs).

  - Rene Rebe <rene@exactcode.de>

--- btrfs-progs-0.12/Makefile.orig	2008-02-10 20:15:16.000000000 +0100
+++ btrfs-progs-0.12/Makefile	2008-02-10 20:15:34.000000000 +0100
@@ -1,6 +1,6 @@
 CC=gcc
 AM_CFLAGS = -Wall -fno-strict-aliasing -D_FILE_OFFSET_BITS=64 -D_FORTIFY_SOURCE=2
-CFLAGS = -g -Werror
+CFLAGS = -g
 objects = ctree.o disk-io.o radix-tree.o extent-tree.o print-tree.o \
 	  root-tree.o dir-item.o hash.o file-item.o inode-item.o \
 	  inode-map.o crc32c.o rbtree.o extent-cache.o extent_map.o \
